.code-samples.meilisearch.yaml

```diff
@@ -473,3 +473,15 @@ security_guide_delete_key_1: |-
 authorization_header_1: |-
   client = Client('http://127.0.0.1:7700', 'masterKey')
   client.get_keys()
+tenant_token_guide_generate_sdk_1: |-
+  api_key = 'B5KdX2MY2jV6EXfUs6scSfmC...'
+  expires_at = datetime(2025, 12, 20)
+  search_rules = {
+    'patient_medical_records': {
+      'filter': 'user_id = 1'
+    }
+  }
+  token = client.generate_tenant_token(search_rules=["*"], api_key=api_key, expires_at=expires_at)
```

Comment:
@alallema you defined the `search_rules` variable but you used `["*"]` in the `generate_tenant_token` call :)